app/build.gradle

```diff
@@ -3,8 +3,8 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion "27.0.3"
+    compileSdkVersion 28
+    buildToolsVersion '28.0.2'
```

Comment:
you'll need to update `.travis.yml` too

Reply:
Updated travis config to match. Tests are green now.